packages/@react-native-windows/cli/src/e2etest/__snapshots__/projectConfig.test.ts.snap

```diff
@@ -0,0 +1,154 @@
+  "project": Object {
+    "projectFile": "SimpleCSharpApp\\\\SimpleCSharpApp.csproj",
+    "projectGuid": "{416476D5-974A-4EE2-8145-4E331297247E}",
+    "projectLang": "cs",
+    "projectName": "SimpleCSharpApp",
+  },
+  "solutionFile": "SimpleCSharpApp.sln",
+  "sourceDir": "windows",
+  "useWinUI3": false,
+}
+`;
+
+exports[`projectConfig - SimpleCppApp (Ignore react-native.config.js) 1`] = `
+Object {
+  "folder": StringContaining "SimpleCppApp",
+  "project": Object {
+    "projectFile": "SimpleCppApp\\\\SimpleCppApp.vcxproj",
+    "projectGuid": "{416476d5-974a-4ee2-8145-4e331297247e}",
+    "projectLang": "cpp",
+    "projectName": "SimpleCppApp",
+  },
+  "solutionFile": "SimpleCppApp.sln",
+  "sourceDir": "windows",
+  "useWinUI3": false,
+}
+`;
+
+exports[`projectConfig - SimpleCppApp (Use react-native.config.js) 1`] = `
+Object {
+  "folder": StringContaining "SimpleCppApp",
+  "project": Object {
+    "projectFile": "SimpleCppApp\\\\SimpleCppApp.vcxproj",
+    "projectGuid": "{416476d5-974a-4ee2-8145-4e331297247e}",
+    "projectLang": "cpp",
+    "projectName": "SimpleCppApp",
+  },
+  "solutionFile": "SimpleCppApp.sln",
+  "sourceDir": "windows",
+  "useWinUI3": false,
+}
+`;
+
+exports[`projectConfig - WithWinUI3 (Ignore react-native.config.js) 1`] = `
+Object {
+  "folder": StringContaining "WithWinUI3",
+  "project": Object {
+    "projectFile": "WithWinUI3\\\\WithWinUI3.vcxproj",
+    "projectGuid": "{416476d5-974a-4ee2-8145-4e331297247e}",
+    "projectLang": "cpp",
+    "projectName": "WithWinUI3",
+  },
+  "solutionFile": "WithWinUI3.sln",
+  "sourceDir": "windows",
+  "useWinUI3": false,
+}
+`;
+
+exports[`projectConfig - WithWinUI3 (Use react-native.config.js) 1`] = `
+Object {
+  "folder": StringContaining "WithWinUI3",
+  "project": Object {
+    "projectFile": "WithWinUI3\\\\WithWinUI3.vcxproj",
+    "projectGuid": "{416476d5-974a-4ee2-8145-4e331297247e}",
+    "projectLang": "cpp",
+    "projectName": "WithWinUI3",
+  },
+  "solutionFile": "WithWinUI3.sln",
+  "sourceDir": "windows",
+  "useWinUI3": true,
+}
+`;
+
+exports[`useWinUI3=true in react-native.config.js, useWinUI3=false in ExperimentalFeatures.props 1`] = `
+"<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
+<packages>
+  <package id=\\"Microsoft.ReactNative.Cxx\\" version=\\"1.0.0\\" targetFramework=\\"native\\"/>
+  <package id=\\"Microsoft.WinUI\\" version=\\"3.0.0-preview3.201113.0\\" targetFramework=\\"native\\"/>
+</packages>"
+`;
+
+exports[`useWinUI3=true in react-native.config.js, useWinUI3=false in ExperimentalFeatures.props 2`] = `
+"<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
+<Project xmlns=\\"http://schemas.microsoft.com/developer/msbuild/2003\\">
+
+  <PropertyGroup Label=\\"Microsoft.ReactNative Experimental Features\\">
+    
+    <!--
+      Changes compilation to assume use of WinUI 3 instead of System XAML.
+      Requires creation of new project.
+
+      See https://microsoft.github.io/react-native-windows/docs/0.64/winui3
```

Comment:
this should be an evergreen link, not version specific

Reply:
The reason I chose not to use an evergreen link here is because what is experimental can change between versions. It's possible someone could click an old version, and the docs no longer exist on latest